docs/faq.rst

```diff
@@ -156,6 +156,12 @@ harder to maintain the code for older Python versions.  If you really need
 Python 2.3 support you either have to use `Jinja 1`_ or other templating
 engines that still support 2.3.
 
+Python 2.4/2.5/3.1/3.2 support was removed when we switched to supporting
+Python 2 and 3 by the same sourcecode (without using 2to3). It was required to
+drop support because only Python 2.6/2.7 and >=3.3 support byte and unicode
+literals in an easy and pretty way. If you really need support for older
```

Comment:
That phrasing is actually incorrect... every Python version has some form of byte and unicode literals, but in the versions supported by Jinja, the syntax overlaps.